content/en/docs/setup/install/virtual-machine/index.md

```diff
@@ -168,7 +198,8 @@ Run the following commands on the virtual machine you want to add to the Istio m
 
 1. Securely transfer the files from `"${WORK_DIR}"/"${CLUSTER_NAME}"/"${SERVICE_NAMESPACE}"`
     to the virtual machine.  How you choose to securely transfer those files should be done with consideration for
-    your information security policies.
+    your information security policies. For convenience in this guide we think you transfer all the required files 
```

Comment:
`For convenience in this guide, you should transfer all of the required files to "${HOME}" in the virtual machine.